code/modules/admin/topic.dm

```diff
@@ -4583,3 +4584,219 @@
+						rel_added.holy_book.item_state = "melted"
+					if("Unaussprechlichen Kulten")
+						rel_added.holy_book.icon_state = "kulten"
+						rel_added.holy_book.item_state = "kulten"
+					if("Necronomicon")
+						rel_added.holy_book.icon_state = "necronomicon"
+						rel_added.holy_book.item_state = "necronomicon"
+					if("Book of Shadows")
+						rel_added.holy_book.icon_state = "shadows"
+						rel_added.holy_book.item_state = "shadows"
+					if("Torah")
+						rel_added.holy_book.icon_state = "torah"
+						rel_added.holy_book.item_state = "torah"
+					if("Burning")
+						rel_added.holy_book.icon_state = "burning"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("Honk")
+						rel_added.holy_book.icon_state = "honkbook"
+						rel_added.holy_book.item_state = "honkbook"
+					if("Ianism")
+						rel_added.holy_book.icon_state = "ianism"
+						rel_added.holy_book.item_state = "ianism"
+					if("The Guide")
+						rel_added.holy_book.icon_state = "guide"
+						rel_added.holy_book.item_state = "guide"
+					if("Slab")
+						rel_added.holy_book.icon_state = "slab"
+						rel_added.holy_book.item_state = "slab"
+						rel_added.holy_book.desc = "A bizarre, ticking device... That looks broken."
+					else
+						//If christian bible, revert to default
+						rel_added.holy_book.icon_state = "bible"
+						rel_added.holy_book.item_state = "bible"
+
+				var/msg = "[usr] created a religion: [rel_added.name]."
+				msg = "<span class=\"admin\"><span class=\"prefix\">ADMIN LOG:</span> <span class=\"message\">[msg]</span></span>"
+				log_adminwarn(msg)
+
+				ticker.religions += rel_added
+				updateRelWindow()
+			if ("delete")
+				if (!href_list["rel"])
+					return FALSE
+
+				var/datum/religion/R = locate(href_list["rel"])
+
+				if (!R || !istype(R, /datum/religion))
+					return FALSE
+
+				if (R.adepts.len)
+					to_chat(usr, "<span class='warning'>You can't delete a religion which has adepts.</span>")
+					return FALSE
+
+				var/msg = "[usr] deleted a religion: [R.name]."
+				ticker.religions -= R
+				qdel(R.holy_book)
+				qdel(R)
+				msg = "<span class=\"admin\"><span class=\"prefix\">ADMIN LOG:</span> <span class=\"message\">[msg]</span></span>"
+				log_adminwarn(msg)
+				updateRelWindow()
+
+			if ("activate")
+				if (!href_list["rel"])
+					return FALSE
+
+				var/datum/religion/R = locate(href_list["rel"])
+
+				if (!R || !istype(R, /datum/religion))
+					return FALSE
+
+				if (R.adepts.len)
+					to_chat(usr, "<span class='warning'>The religion already has adepts!</span>")
+					return FALSE
+
+				if (alert("Do you wish to activate this religion? You will have to pick a player as its guide. Make sure the player is aware your plans!", "Activating a religion", "Yes", "No") != "Yes")
+					return FALSE
+
+				var/mob/living/carbon/human/preacher = input(usr, "Who should be the leader of this new religion?", "Activating a religion") as mob in world // THERE HAS TO BE A BETTER WAY TO DO THAT
```

Comment:
DEFINITELY
you could try ` in sortmobs()` or maybe filter the output of `sortmobs()` to exclude those without a ckey first